lib/phoenix_swagger/json_api.ex

```diff
@@ -249,7 +250,7 @@ defmodule PhoenixSwagger.JsonApi do
             }
           },
           data: %Schema{
-            type: :object,
+            type: opts[:type] || :object,
```

Comment:
I'm not sure this will produce valid swagger? When `type` is `array`, it should use `items` to specify the array items schema. When type is `object` it should use `properties`.